helpers/doesNotNeedPlugin.js

```diff
@@ -0,0 +1,23 @@
+// Checks all the cases for which the plugin should do nothing
+const isStaticExportProject = require('./isStaticExportProject')
+const doesSiteUseNextOnNetlify = require('./doesSiteUseNextOnNetlify')
+const hasCorrectNextConfig = require('./hasCorrectNextConfig')
+
+const doesNotNeedPlugin = ({ netlifyConfig, packageJson, nextConfigPath }) => {
+  const { build } = netlifyConfig
+  const { name, scripts = {}, dependencies = {} } = packageJson
+
+  const hasNoPackageJson = Object.keys(packageJson).length === 0
+  if (hasNoPackageJson) {
+    console.log('Could not find a package.json for this project')
```

Comment:
Should we keep this as a failure?

Reply:
that sounds like a solid idea erez thanks! will implement :) @ true/false

and yeah we can keep it is a failure!